app/views/meetings/show.html.erb

```diff
@@ -44,7 +44,10 @@ See doc/COPYRIGHT.md for more details.
   <% end %>
   <% if authorize_for(:meetings, :destroy) %>
     <li class="toolbar-item">
-      <%= link_to({:controller => '/meetings', :action => 'destroy', :id => @meeting}, class: 'button') do %>
+      <%= link_to({controller: '/meetings', action: 'destroy', id: @meeting},
+                  class: 'button',
+                  method: :delete,
+                  confirm: l(:text_are_you_sure)) do %>
```

Comment:
Yes, let's wait for opf/openproject#3029. Otherwise, the relative simple fix get's bigger and bigger, as there are other links above the fixed one that also do not use named routes yet. Additionally, I might not have to do it that way :)